qiskit_experiments/calibration/calibration_types.py

```diff
@@ -0,0 +1,23 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Types used by the calibration module."""
```

Comment:
To me `calibration_types` sounds like type of calibration, such as amplitude cal, frequency cal, etc... What do you think renaming this to something like `common_types`?

Reply:
Renamed to `calibration_key_types` see here: c1e29db